core/command/index.js

```diff
@@ -20,7 +20,8 @@ var commandNames = [
   'test',
   'approve',
   'version',
-  'stop'
+  'stop',
+  'engineErrors'
```

Comment:
I think you should be able to just add this as a regular module (from `/core/util`).  We should avoid creating this as a command since this would never be run alone.